source/_docs/scripts.markdown

```diff
@@ -610,6 +610,63 @@ automation:
 
 {% endraw %}
 
+
+More `choose` can be used together. This is the case of an IF-IF.
+
+{% raw %}
+
+```yaml
+automation:
```

Comment:
Please adjust the example to comply with our YAML style guide.

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

Reply:
Done it, must be correct now